packages/components/src/components/button/button.scss

```diff
@@ -84,20 +80,20 @@
             }
         }
         &--red {
-            background: var(--button-primary-default);
+            background: var(--status-danger);
```

Comment:
we won't use --red anymore, we will only use button primary, secondary, or tertiary style button. and the red color in the button primary is `var(--button-primary-default)` color